appveyor.yml

```diff
@@ -1,11 +1,13 @@
 # Generated from:
 # https://github.com/zopefoundation/meta/tree/master/config/c-code
+# Currently the builds use @dataflake's Appveyor account.  The PyPI token belongs
+# to zope.wheelbuilder, which is managed by @mgedmin and @dataflake.
```

Comment:
This comment will be lost since it's not in .meta.toml.

In zope.interface I put it inside the global-env-vars section for simplicity: https://github.com/zopefoundation/zope.interface/pull/244/files

Reply:
Done on master